deploy/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+cassandra:
+  image: itszero/zipkin-cassandra
+collector:
+  image: itszero/zipkin-collector
+  expose:
+    - 9410
+  ports:
+    - 9410:9410
+  links:
+    - cassandra:db
+query:
+  image: itszero/zipkin-query
+  expose:
+    - 9411
+  links:
+    - cassandra:db
+web:
+  image: itszero/zipkin-web
+  links:
+    - query
+  ports:
+    - 8080:8080
+  environment: {
+    ROOTURL: "http://172.17.42.1:8080"
```

Comment:
@zsol I don't know why the `ROOTURL` environment variable is needed for `zipkin-web`. If I remove it everything still works. Also it is a hard coded private ip address...  I guess we should remove it.

Can you also try if everything still works in your environment when you remove it?